organization_token.go

```diff
@@ -64,6 +74,27 @@ func (s *organizationTokens) Create(ctx context.Context, organization string) (*
 	return ot, err
 }
 
+// CreateWithOptions a new organization token with options, replacing any existing token.
+func (s *organizationTokens) CreateWithOptions(ctx context.Context, organization string, options OrganizationTokenCreateOptions) (*OrganizationToken, error) {
```

Comment:
Suggestion: `Create` pretty much duplicates this. I think we could update `Create` to just call this function, and pass in the default `OrganizationTokenCreateOptions{}`